cmd/zetatool/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/zetacore/cmd/zetatool/config"
+	"github.com/zeta-chain/zetacore/cmd/zetatool/filterdeposit"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "zetatool",
+	Short: "utility tool for zeta-chain",
+}
+
+func init() {
+	rootCmd.AddCommand(filterdeposit.Cmd)
+	rootCmd.PersistentFlags().String(config.Flag, "", "custom config file: --config filename.json")
```

Comment:
This is the config for filtering (it would be too unclean to have a general config for all commands we may have) we should us a local flag instead of persistent for the `filterdeposit` command

Reply:
The config will be more generic in the next update, I'd like to have one config for several tools under zetatool_config.json